packages/vite-plugin/src/index.ts

```diff
@@ -56,15 +58,17 @@ export function vanillaExtractPlugin({ identifiers }: Options = {}): Plugin {
       packageInfo = getPackageInfo(config.root);
     },
     resolveId(id) {
-      if (id.indexOf(virtualExt) > 0) {
+      if (id.startsWith(virtualPrefix) && id.indexOf(virtualExt) > 0) {
```

Comment:
Think we can drop the virtualExt check now as it's basically redundant. Also we prefer to use `indexOf` instead of `startsWith` as it seems to be quite a lot more performant.

Reply:
Done these, too.